content/en/docs/setup/independent/kubelet-integration.md

```diff
@@ -193,10 +193,10 @@ The DEB and RPM packages shipped with the Kubernetes releases are:
 
 | Package name | Description |
 |--------------|-------------|
-| `kubeadm`    | Installs the `/usr/bin/kubeadm` CLI tool and [The kubelet drop-in file(#the-kubelet-drop-in-file-for-systemd) for the kubelet. |
+| `kubeadm`    | Installs the `/usr/bin/kubeadm` CLI tool and [the kubelet drop-in file](#the-kubelet-drop-in-file-for-systemd) for the kubelet. |
```

Comment:
`[the kubelet drop-in file]` -> `the [kubelet drop-in file]`

Reply:
done